Montuj to na Maina szefie

MKamilAmin opened the pull request:
Dodane:
-Modal
-Stylizacja scroll-baru
-Favicon
-Wstępna funkcjonalność Library
-API w stylu object oriented programming (osobny plik)
-Oraz wiele innych i to wszystko za jedyne 15,99 zł

MKamilAmin committed:
Feature(HTML) Added modal functionality

Added Modal building structure and lincked modal html to index.html and created class based API fetch system. Also added favicon.

MKamilAmin committed:
Squashed commit of the following:

commit 06d920543a69422fb2febe599861479537154b9e
Merge: d41ec80 8ecd505
Author: anitabaron <111062951+anitabaron@users.noreply.github.com>
Date:   Thu Aug 1 18:05:33 2024 +0200

    Merge pull request #22 from anitabaron/hero

commit 8ecd505bb0f750c373d468774e0ed5db4dc4103b
Merge: bea1fcb d41ec80
Author: anitabaron <111062951+anitabaron@users.noreply.github.com>
Date:   Thu Aug 1 18:02:40 2024 +0200

    Merge branch 'main' into hero

commit d41ec801dbcd73b3ceabfb486e80dcc7df71e5f8
Author: AlicjaSzamraj <120415960+AlicjaSzamraj@users.noreply.github.com>
Date:   Thu Aug 1 16:55:21 2024 +0100

    Icons (#21)

commit 2442997119308faef645322cd74b7bdf3889e73f
Author: DorotaDobija <117933735+DorotaDobija@users.noreply.github.com>
Date:   Thu Aug 1 17:39:07 2024 +0200

    Movie list changes (#20)

    * genre fuction and release year added

    * movie-list-changes

    * new func added to fetchapi

commit 2a70b9dab375e1a421509bc29d69941f4e050503
Merge: 67f04b6 97742bb
Author: KrystianRyczek <153994194+KrystianRyczek@users.noreply.github.com>
Date:   Thu Aug 1 15:45:20 2024 +0200

    Merge pull request #23 from anitabaron/catalog---statup

    Catalog-startup

commit 97742bbf26e0c64a8b43ad38bbec0262e5419ac6
Author: KrystianRyczek <153994194+KrystianRyczek@users.noreply.github.com>
Date:   Thu Aug 1 15:34:16 2024 +0200

    Catalog-startup

    fetch data on start up

commit bea1fcbe2e9fc5533ed84e5edb96f1c920619198
Author: amper00 <141366407+amper00@users.noreply.github.com>
Date:   Thu Aug 1 11:49:02 2024 +0200

    gwiazdki do wglądu

commit b5e130586a838c8968b3f1ed9282e20cb21c145b
Author: amper00 <141366407+amper00@users.noreply.github.com>
Date:   Thu Aug 1 10:17:40 2024 +0200

    try

commit 525ef98c8200677edfc09f0605ce5f0f94b31eac
Author: amper00 <141366407+amper00@users.noreply.github.com>
Date:   Thu Aug 1 00:52:28 2024 +0200

    try

MKamilAmin committed:
Feat - Added library and storage functions

MKamilAmin committed:
Merge branch 'main' into modal-kamil

MKamilAmin committed:
Merge branch 'main' into modal-kamil

MKamilAmin committed:
Feat - Added modal

-Dodałem modal
-Dodałem favicon
-Ustylizowałem scroll-bar
-Dodałem wstępną funkcjonalność biblioteki

MKamilAmin committed:
Merge branch 'main' into modal-kamil

MKamilAmin committed:
Usunąłem stary plik

przedostał się przez przypadek jak mergowałem lokalnie.

anitabaron committed:
Update movie-list.scss

remove card list justify-content

anitabaron committed:
Update movie-list.scss

anitabaron commented on src/images/icons_desktop.svg:
svg nie sprawdzam, bo wiem że to ogarniasz a ja nie wiem co tam było nie tak

anitabaron commented on src/js/index.js:
```diff
@@ -8,3 +8,4 @@ import './modal';
 import './oops';
 import './buttons-icons';
 import './api-home';
+import './api-catalog-page';
```
import './api-catalog-page';  
nie może być w index.js bo to jest do catalogu tylko, wywalam to przez edit

anitabaron committed:
Update index.js

import './api-catalog-page';
nie może być w index.js bo to jest do catalogu tylko, wywalam to przez edit

anitabaron commented on src/js/cards-rendering.js:
```diff
@@ -0,0 +1,163 @@
+import { emptyStar, fullStar, halfStar } from './stars';
```
a czemu tu jest card rendering jak one się renderują w innym miejscu już i tam się tworzą, to jest jedynie do działania w my-library?
no i gwiazdku to Daniel z nimi walczył z nimi i już się super wyświetlały te jego, czemu chcesz wstawić inne?
generalnie chyba to wszystko trafia w endpoint na naszej stronie my-library więc ja to puszczam, ale to jest do skoordynowania i ograniczenia się do jednego z rozwiązań, żebyśmy nie mnożyli tych samych spraw

anitabaron commented on src/js/stars.js:
```diff
@@ -0,0 +1,36 @@
+const emptyStar = `<svg class="star" width="18" height="18" viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg">
```
puszczam to z zastrzeżeniem porównania/przejrzenia z najnowszymi gwiazdkami Daniela które ładnie działają teraz

anitabaron commented:
Conflicting files :
src/js/api-catalog-page.js
biore to przez edycje, tam została dodana paginacja wczoraj wieczorem, postaram się te kody porównać tak żeby obie funckjonalności były zachowane, zobaczymy czy się uda

anitabaron committed:
Merge branch 'main' into modal-kamil

anitabaron commented:
w pliku z konfliktami zostawiłam zakomentowane rzeczy, bo musze za chwile wyjechac i nie dam rady sprawdzić dokładnie , nie chce zrobić błędu żeby się rozsypało
